src/Payment.php

```diff
@@ -51,21 +51,23 @@ class Payment extends Model
     /**
      * @param MolliePayment $payment
      * @param \Illuminate\Database\Eloquent\Model $owner
+     * @param array $actions
      * @param array $overrides
      * @return static
      */
-    public static function createFromMolliePayment(MolliePayment $payment, Model $owner, array $overrides = []): self
+    public static function createFromMolliePayment(MolliePayment $payment, Model $owner, array $actions = [], array $overrides = []): self
     {
-        return tap(static::makeFromMolliePayment($payment, $owner, $overrides))->save();
+        return tap(static::makeFromMolliePayment($payment, $owner, $actions, $overrides))->save();
     }
 
     /**
      * @param MolliePayment $payment
      * @param \Illuminate\Database\Eloquent\Model $owner
+     * @param array $actions
      * @param array $overrides
      * @return static
      */
-    public static function makeFromMolliePayment(MolliePayment $payment, Model $owner, array $overrides = []): self
+    public static function makeFromMolliePayment(MolliePayment $payment, Model $owner, array $actions = [], array $overrides = []): self
```

Comment:
These actions are not coming from the Mollie payment resource, so it's cleaner to use a separate call for setting them. It's an Eloquent model, so this should be ok:

```php
$localPayment = Payment::makeFromMolliePayment($molliePayment, $owner, $overrides);
$localPayment->actions = $actions; // Cast should handle this
$localPayment->save();
```

Reply:
(yeah I know I mentioned adding it to the constructor before, but this makes more sense now I see it ;) )